python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py

```diff
@@ -356,7 +350,7 @@ def predict(self, data, batch_size=None, verbose=1,
             size=self.num_workers,
             model_weights=weights,
             mode="predict",
-            cluster_info=self._get_cluster_info(sc),
+            cluster_info=None,
```

Comment:
Create cluster_info uses barrier mode for get ips and rank.
Since for predict cluster_info is not used, do not call this function, and input None instead.